projects/storefrontstyles/scss/components/product/carousel/_product-carousel.scss

```diff
@@ -1,4 +1,5 @@
-%cx-product-carousel {
+%cx-product-carousel,
+%cx-merchandising-carousel {
```

Comment:
Please create a _merchandising-carousel.scss, and extend the cx-product-carousel if needed, so we can place stylings accordingly to each 'component'

Reply:
Done

Reply:
Similar to what I said above, I know we have been saying cds should be isolated, but the scss should be in our styleslib

Reply:
This comment is on the commit and has been resolved in the latest changes